src/platform/K32W/Logging.cpp

```diff
@@ -21,26 +21,29 @@
 static bool isLogInitialized;
 extern uint8_t gOtLogUartInstance;
 extern "C" void K32WWriteBlocking(const uint8_t * aBuf, uint32_t len);
+extern "C" uint32_t otPlatAlarmMilliGetNow(void);
 
 namespace chip {
 namespace Logging {
 namespace Platform {
 
 void GetMessageString(char * buf, uint8_t chipCategory, uint8_t otLevelLog)
```

Comment:
Can you add a buffer length parameter please?

Reply:
@mspang done. Thanks!